ChronoGrapher/CMakeLists.txt

```diff
@@ -1,27 +1,31 @@
-cmake_minimum_required(VERSION 3.19)
+cmake_minimum_required(VERSION 3.25)
 
 find_package(Thallium REQUIRED)
 find_package(spdlog REQUIRED)
 
+message("Chronographer CMakeLists - CMAKE_SOURCE_DIR=${CMAKE_SOURCE_DIR}")
+message("Chronographer CMakeLists - CMAKE_CURRENT_SOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
`CMAKE_CURRENT_SOURCE_DIR` is printed twice.

Reply:
i do not know exactly what you mean because this is happening in every single CMakeLists.txt file. There is a message for printing `CMAKE_SOURCE_DIR` and another for `CMAKE_CURRENT_SOURCE_DIR` and then building information and target information. 

If I understood everything properly both the building and the message print the `CMAKE_CURRENT_SOURCE_DIR`. I guess that the error. One is for debugging the other is for information. 

Consequently, I commented the two debugging message for `CMAKE_SOURCE_DIR` and `CMAKE_CURRENT_SOURCE_DIR`